content/articles/2018-08-scales-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,98 @@
+---
+title: "scales 1.0.0"
+author: Dana Seidel
+date: '2018-07-30'
+slug: scales-1-0-0
+description: >
+  scales 1.0.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/3kzlCL3rj8A
+  author: Ricardo Gomez Angel
+tags: []
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+We're delighted to announce the release of [scales](https://scales.r-lib.org/) 1.0.0. 
+This is a major release with significant changes to the popular formatter
+functions and added transformations.
```

Comment:
Genuine oxford-comma question moment: are you saying significant changes to the…formatter functions and transformations, or are you saying there are significant changes…and added transformations?

Reply:
the latter, hopefully it's clearer now.